[2025-02-14T10:32:30.385+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T10:32:30.401+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: create_file_dag.create_file_task manual__2025-02-14T10:32:28.549919+00:00 [queued]>
[2025-02-14T10:32:30.412+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: create_file_dag.create_file_task manual__2025-02-14T10:32:28.549919+00:00 [queued]>
[2025-02-14T10:32:30.412+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-14T10:32:30.607+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): create_file_task> on 2025-02-14 10:32:28.549919+00:00
[2025-02-14T10:32:30.615+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=242) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-14T10:32:30.617+0000] {standard_task_runner.py:72} INFO - Started process 244 to run task
[2025-02-14T10:32:30.617+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'create_file_dag', 'create_file_task', 'manual__2025-02-14T10:32:28.549919+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/create_file_dag.py', '--cfg-path', '/tmp/tmply2ff4ip']
[2025-02-14T10:32:30.618+0000] {standard_task_runner.py:105} INFO - Job 40: Subtask create_file_task
[2025-02-14T10:32:30.719+0000] {task_command.py:467} INFO - Running <TaskInstance: create_file_dag.create_file_task manual__2025-02-14T10:32:28.549919+00:00 [running]> on host 7b78e6c4b0cc
[2025-02-14T10:32:30.773+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='create_file_dag' AIRFLOW_CTX_TASK_ID='create_file_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-14T10:32:28.549919+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-14T10:32:28.549919+00:00'
[2025-02-14T10:32:30.775+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-14T10:32:30.775+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-14T10:32:30.776+0000] {logging_mixin.py:190} INFO - Current task name:create_file_task state:running start_date:2025-02-14 10:32:30.402241+00:00
[2025-02-14T10:32:30.777+0000] {logging_mixin.py:190} INFO - Dag name:create_file_dag and current dag run status:running
[2025-02-14T10:32:30.777+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-14T10:32:30.779+0000] {logging_mixin.py:190} INFO - 檔案 /home/***/1 已經創建！
[2025-02-14T10:32:30.779+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-14T10:32:30.785+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-14T10:32:30.786+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=create_file_dag, task_id=create_file_task, run_id=manual__2025-02-14T10:32:28.549919+00:00, execution_date=20250214T103228, start_date=20250214T103230, end_date=20250214T103230
[2025-02-14T10:32:30.803+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-14T10:32:30.804+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-14T10:32:30.804+0000] {logging_mixin.py:190} INFO - Dag name:create_file_dag queued_at:2025-02-14 10:32:28.566711+00:00
[2025-02-14T10:32:30.804+0000] {logging_mixin.py:190} INFO - Task hostname:7b78e6c4b0cc operator:PythonOperator
[2025-02-14T10:32:30.832+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-14T10:32:30.843+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-14T10:32:30.846+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
